app/services/invite_schools.rb

```diff
@@ -274,6 +274,18 @@ def invite_sitless_not_opted_out_schools_for_nqt_plus_one
     end
   end
 
+  def inform_diy_schools_of_wordpress
+    User.where(
+      id: diy_school_cohorts_without_pending_partnerships
+            .joins(school: :induction_coordinators)
+            .pluck(:user_id),
+    ).distinct.find_each do |user|
```

Comment:
No need for distinct - there's no join on the top relation.

Reply:
Of course
```suggestion
    ).find_each do |user|
```